frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -6,13 +6,15 @@ import Riders from './Riders';
 import Home from './Home';
 import Locations from './Locations';
 import Settings from './Settings';
+import DriverDetail from '../../components/DriverDetail/DriverDetail';
 
 const Dashboard = () => (
   <Router basename="/dashboard">
     <Sidebar>
       <Switch>
         <Route path="/home" component={Home} />
-        <Route path="/drivers" component={Drivers} />
+        <Route exact path="/drivers/driver" component={DriverDetail} />
```

Comment:
Might be worth creating another router with 2 routes inside the `Drivers` component, one route to show the `/drivers` content and another to show the `/drivers/driver` content. So like a router with basename = "/drivers" and the "/drivers" has router path "/" and the driver specific route has path "/driver"

Reply:
This will also help us avoid using exact path